src/context.rs

```diff
@@ -193,7 +194,7 @@ impl ScheduleContexts {
             ParamBuilder,
             ParamBuilder,
             QueryParamBuilder::new(|builder| {
-                builder.optional(|builder| {
+                builder.without::<ExternallyMocked>().optional(|builder| {
```

Comment:
Why disable it for the conditions and modifiers query? If the action is skipped, it won't be used anyway.

Reply:
you're right